### PR TITLE
Add an option to embed sources for compiled views (cshtml content) in…

### LINKED ISSUE #2: Add an option to embed sources for compiled views (cshtml content) in the generated assembly
We would also need a sample tool that dumps these files back on disk preserving the original file structure.

### PR DESCRIPTION
**Pull request by pranavkm**
… the generated assembly

Fixes #2

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @NTaylorMullen

File: TestApps.sln [hunk 1]
```diff
@@ -19,6 +19,8 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "ApplicationWithConfigureMvc
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "ClassLibraryWithPrecompiledViews", "testapps\ClassLibraryWithPrecompiledViews\ClassLibraryWithPrecompiledViews.xproj", "{4684DE8B-3FBE-421B-8798-56C3D6698B76}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "PublishWIthEmbedViewSources", "testapps\PublishWIthEmbedViewSources\PublishWIthEmbedViewSources.xproj", "{E3462190-3068-40F0-9AA5-34779FE252AC}"
```
Comment (NTaylorMullen) [inline TestApps.sln]: `WIth` lawlz

File: test/Microsoft.AspNetCore.Mvc.Razor.Precompilation.FunctionalTests/PublishWithEmbedViewSourcesTest.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        private const string ApplicationName = "PublishWithEmbedViewSources";
+
+        public PublishWithEmbedViewSourcesTest(PublishWithEmbedViewSourcesTestFixture fixture)
+        {
+            Fixture = fixture;
+        }
+
+        public ApplicationTestFixture Fixture { get; }
+
+        public static IEnumerable<object[]> SupportedFlavorsTheoryData
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNetCore.Mvc.Razor.Precompilation.FunctionalTests/PublishWithEmbedViewSourcesTest.cs]: Should just use TheoryData

**Comment by NTaylorMullen** [comment]
:shipit:

**Pull request closed without merging**